packages/shared/lib/helpers.ts

```diff
@@ -251,3 +251,21 @@ export function deepCopy(obj: unknown): unknown {
         }, {})
```

Comment:
We can do this in the major refactor coming, but we should have separate helper files for different topics under a helper folder, i.e. this is related to colours.